src/styles/blocks/catalog.scss

```diff
@@ -14,6 +14,14 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    cursor: pointer;
+
+    transition: $transition-time;
```

Comment:
You should write which parameters will use your transition.
For example: `transition: color 300ms`